dali/operators/reader/loader/webdataset_loader.h

```diff
@@ -39,6 +39,10 @@ const std::set<DALIDataType> kSupportedTypes = {DALI_UINT8,   DALI_UINT16, DALI_
                                                 DALI_INT8,    DALI_INT16,  DALI_INT32,  DALI_INT64,
                                                 DALI_FLOAT16, DALI_FLOAT,  DALI_FLOAT64};
 
+constexpr int MakeVersion(int major, int minor) {
```

Comment:
How about using `GetVersionNumber` instead?

Reply:
That would sound like it _extracts_ something, not constructs something. The idea was that it's MakeVersion, but GetMajor, GetMinor. If you insist, I can change, but I get this vibe that get is "get Y from X", not "construct Z from X and Y".

Reply:
I mean to use the function we already have in `core/util.h`. Feel free to rename it but it would be good to avoid creating another one.

Reply:
OK, thanks for pointing me to that utility - I didn't know about its existence (and, honestly, didn't expect it in something as generic as core/util.h). I'll probably move them to a separate header (version_util?). Arguably they'll be easier to find this way.

Reply:
I'm fine either way. Originally it was created for generating integer versions for libraries we load dynamically now (npp, cuFF, nvJPEG). The `util.h` seemed a good place for that as I didn't want to create another file just for that (I guess due to my laziness).